bodhi/client/__init__.py

```diff
@@ -661,13 +661,8 @@ def print_resp(resp, client):
     elif 'comment' in resp:
         click.echo('The following comment was added to %s' % resp.comment['update'].title)
         click.echo(resp.comment.text)
-    elif 'errors' in resp:
-        for error in resp['errors']:
-            click.echo("ERROR: %s" % error['description'])
-        sys.exit(1)
```

Comment:
At first this made me nervous, but I realized that we already have code that handles this in the bindings so this would never happen:

https://github.com/fedora-infra/bodhi/blob/2.11.0/bodhi/client/bindings.py#L92-L101

Reply:
Sorry! i meant to mention this in the PR description. :)

I was a little hard to write a test for something that we would never hit :D